backbone/rolodex_exercise_steps/step4/instructions.html

```diff
@@ -16,7 +16,8 @@
 
 	<h2>Setup</h2>
 	<ol>
-        <li>If you are not yet running your code from a local server, do that now by running <code>python -m SimpleHTTPServer</code> from your directory. We need to do this because of security restrictions on file:// protocol with XMLHttpRequests.</li>
+        <li>If you are not yet running your code from a local server, do that now by following 
+      <a href="../resources/testinglocally.html" target="_blank">these instructions</a>. We need to do this because of security restrictions on file:// protocol with XMLHttpRequests.</li>
```

Comment:
I think these need at least one more `../` in the URL. @verythorough can you test and update?